woocommerce/woocommerce-gutenberg-products-block
# i18n: Remove the babel makepot & pot-to-php scripts

These scripts aren’t needed because WP.org will now pull from JS files for GlotPress translations. See #339.

The purpose of these scripts was to generate a pot file from the JS files, and then compile that into a PHP file that only existed to put the strings into a format that WordPress.org would understand. Now that wp.org can pull from JS files, we don't need to do these steps.

### How to test the changes in this Pull Request:

1. Run `npm run build`
2. Expect: It should not generate a `languages/woo-gutenberg-products-block.php` file
